llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/llama_index/embeddings/ipex_llm/base.py

```diff
@@ -83,11 +84,10 @@ def __init__(
             **model_kwargs,
         )
 
-        from ipex_llm.transformers.convert import _optimize_pre, _optimize_post
```

Comment:
Is my understanding correct in that there is no real change to core logic in this code base. And so, this PR is mostly about showcasing new examples?

Reply:
Thank you for the comment :)

For `IPEXLLMEmbedding`, we do have the initial code support in the previous PR, but did not make it available to users due to no installation related support (e.g. `pyproject.toml`, examples, etc.).

In this PR, we updated the dependency for running on Intel XPU, and conducted more verification as well as fixes some bug in the code to make Intel GPU support available to users.